### ISSUE TITLE
Error trying to move a SQL instance to another region/zone

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#358]
```
Issue running command [gcloud beta sql instances patch].

What steps will reproduce the problem?

$ gcloud beta sql instances patch heatwave --gce-zone asia-east1
The following message will be used for the patch API method.
{"project": "stormy-weather-001", "name": "heatwave", "settings": 
{"locationPreference": {"zone": "asia-east1"}}}
WARNING: This patch modifies the zone your instance is set to run in, 
which may require it to be moved. Submitting this patch will restart 
your instance if it is running in a different zone.

Do you want to continue (Y/n)?  y

What is the expected output? What do you see instead?

EXPECTED: To have my instance moved to a different zone.

INSTEAD:



ERROR: gcloud crashed (TypecheckError): Type of arg is "<class 
'googlecloudsdk.third_party.apis.sqladmin.v1beta4.sqladmin_v1beta4_messages.Data
baseInstance'>", not "<class 
'googlecloudsdk.third_party.apis.sqladmin.v1beta4.sqladmin_v1beta4_messages.SqlI
nstancesPatchRequest'>"


Please provide any additional information below.


Traceback (most recent call last):
  File "calliope/cli.py", line 598, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "calliope/backend.py", line 1226, in Run
    resources = command_instance.Run(args)
  File "api_lib/sql/errors.py", line 28, in Func
    return foo(*args, **kwargs)
  File "surface/sql/instances/patch.py", line 321, in Run
    result_operation = sql_client.instances.Patch(patch_instance)
  File "third_party/apis/sqladmin/v1beta4/sqladmin_v1beta4_client.py", line 654, in Patch
    config, request, global_params=global_params)
  File "third_party/apitools/base/py/base_api.py", line 621, in _RunMethod
    method_config, request, global_params, upload, upload_config, download)
  File "third_party/apitools/base/py/base_api.py", line 583, in PrepareHttpRequest
    util.Typecheck(request, request_type)
  File "third_party/apitools/base/py/util.py", line 75, in Typecheck
    raise exceptions.TypecheckError(msg)
TypecheckError: Type of arg is "<class 
'googlecloudsdk.third_party.apis.sqladmin.v1beta4.sqladmin_v1beta4_messages.Data
baseInstance'>", not "<class 
'googlecloudsdk.third_party.apis.sqladmin.v1beta4.sqladmin_v1beta4_messages.SqlI
nstancesPatchRequest'>"

Installation information:

Google Cloud SDK [0.9.84]

Platform: [Linux, x86_64]
Python Version: [2.7.6 (default, Jun 22 2015, 17:58:13)  [GCC 4.8.2]]
Site Packages: [Disabled]

Installation Root: [/usr/local/google/home/meirellesl/google-cloud-sdk]
Installed Components:
  core: [2015.10.23]
  app-engine-python: [1.9.27]
  core-nix: [2015.09.03]
  kubectl: []
  app: [2015.10.23]
  gcloud: [2015.10.23]
  gsutil-nix: [4.14]
  beta: [2015.10.08]
  gsutil: [4.15]
  bq: [2.0.18]
  alpha: [2015.10.08]
  kubectl-linux-x86_64: [1.0.6]
  app-engine-java: [1.9.27]
  bq-nix: [2.0.18]
System PATH: 
[/usr/local/google/home/meirellesl/google-cloud-sdk/bin:/usr/local/google/home/m
eirellesl/google-cloud-sdk/bin:/usr/local/google/home/meirellesl/bin:/usr/lib/go
ogle-golang/bin:/usr/local/buildtools/java/jdk/bin:/usr/local/sbin:/usr/local/bi
n:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/google/home/meirellesl/gsutil:~/bin:/
usr/local/google/home/meirellesl/gsutil:~/bin]
Cloud SDK on PATH: [True]

Installation Properties: 
[/usr/local/google/home/meirellesl/google-cloud-sdk/properties]
User Config Directory: [/usr/local/google/home/meirellesl/.config/gcloud]
User Properties: [/usr/local/google/home/meirellesl/.config/gcloud/properties]
Current Workspace: [None]
Workspace Config Directory: [None]
Workspace Properties: [None]

Account: [meirellesl@google.com]
Project: [stormy-weather-001]

Current Properties:
  [core]
    project: [stormy-weather-001]
    account: [meirellesl@google.com]
    disable_usage_reporting: [False]
  [app]
    use_appengine_api: [True]

Logs Directory: [/usr/local/google/home/meirellesl/.config/gcloud/logs]
Last Log File: 
[/usr/local/google/home/meirellesl/.config/gcloud/logs/2015.10.29/17.54.53.17061
9.log]



```

Original issue reported on code.google.com by `meirell...@google.com` on 29 Oct 2015 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. We are looking into it.
```

Original comment by `gsfow...@google.com` on 3 Nov 2015 at 8:32

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This issue is fixed in Cloud SDK version 0.9.88, to be released this week.
```

Original comment by `z...@google.com` on 24 Nov 2015 at 7:32
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
0.9.88 is out. Please let us know if you can still reproduce this issue.
```

Original comment by `z...@google.com` on 24 Nov 2015 at 9:28
- Changed state: **Fixed**
